http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -220,7 +217,8 @@ class LeonardoService(
         internalCreateCluster(userInfo.userEmail, serviceAccountInfo, googleProject, clusterName, clusterRequest)
       )(c => IO.raiseError(ClusterAlreadyExistsException(googleProject, clusterName, c.status)))
 
-    } yield cluster
+      runtimeConfig <- Queries.getRuntime(cluster.runtimeConfigId).transaction
```

Comment:
Can we make `internalCreateCluster` return the `RuntimeConfig` instead of querying for it from the DB? Trying to keep our response times low.

Reply:
this db call isn't actually necessary, I'll remove it